index.html

```diff
@@ -0,0 +1,233 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Nova</title>
+    <link rel="stylesheet" href="./styles/reset.css" />
+    <link rel="stylesheet" href="./styles/style.css" />
+    <style>
+      @import url("https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100..900;1,100..900&display=swap");
+    </style>
+  </head>
+  <body>
+    <section class="section section__top">
+      <div class="section__container">
+        <nav class="section__navbar navbar">
+          <div class="navbar__logo">
+            <object data="./images/logo.svg"></object>

```

Comment:
Jeśli używasz tylko `svg` jako obrazek (nie ma tam dodatkowej zawartości, tylko sam obraz) to raczej preferowałbym `<img/>`